pkg/continuousprofiling/profiler.go

```diff
@@ -77,7 +100,7 @@ func validateServerAddress(address string) (string, error) {
 	// We default to https unless http is specified
 	sanitizedAddress := urlfmt.FormatURL(address, urlfmt.HTTPS, urlfmt.NoTrailingSlash)
 	if _, err := url.Parse(sanitizedAddress); err != nil {
-		return "", errors.Wrapf(err, "unable to parse the server address %q", address)
+		return "", errors.Wrap(ErrUnableToParseServerAddress, errors.Wrapf(err, "server address %q", address).Error())
```

Comment:
This change reverses the error chain. Was it intended?
Before: `unable to parse the server address "some address": some error`
After: `server address "some address": some error: unable to parse server address`
What about the following:
```
return "", errox.InvalidArgs.Newf("unable to parse server address %q", address).CausedByf(err)
```
Which will give `unable to parse server address "some address": some error`, and will match `ErrorIs(e, errox.InvalidArgs)`.
Alternatively, with one more indirection:
```
ErrUnableToParseServerAddress = errox.InvalidArgs.New("parsing error")
...
return "", ErrUnableToParseServerAddress.Newf("unable to parse server address %q", address).CausedByf(err)
```

Reply:
Good catch! Yeah that was not intentional. I really like you're suggestion:
```
return "", errox.InvalidArgs.Newf("unable to parse server address %q", address).CausedByf(err)
```
I've implemented that in the last commit 🙂

Reply:
Super, thanks. Just one more little improvement :)